services/asset/nodes/requests.go

```diff
@@ -0,0 +1,53 @@
+package nodes
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/leanspace/terraform-provider-leanspace/provider"
+)
+
+type apiShiftNodeInfo struct {
+	TargetParentNodeId string `json:"targetParentNodeId"`
+}
+
+func (node *Node) toAPIFormat() ([]byte, error) {
+	shiftNode := apiShiftNodeInfo{
+		TargetParentNodeId: node.ParentNodeId,
+	}
+	return json.Marshal(shiftNode)
+}
+
+func nodeAction(action string, nodeId string, node *Node, client *provider.Client) error {
+
```

Comment:
nitpicking: this extra line is not needed

Reply:
Done